src/js/no-react-entry.js

```diff
@@ -17,3 +17,7 @@ require('./legacy/mega-menu.js');
 require('./legacy/sidebar-navigation.js');
 
 require('./login/login-entry.jsx');
+
+if (location.href.endsWith('education/apply/')) {
+  require('./education-wizard.js');
```

Comment:
This will keep the code from executing on non-edu pages, but it won't make a separate bundle. However, the file is only 65 lines, I'm not sure it's really worth it to split.